File: JAVA/bubble_sort.java
import java.util.*;
class bubble_sort
{
    public static void bubbleSort(int a[])
    {
        int n=a.length;
        for(int x=0;x<n-1;x++)
        {
            for(int y=0;y<n-x-1;y++)
            {
                if(a[y]>a[y+1])
                {
                    int t=a[y];
                    a[y]=a[y+1];
                    a[y+1]=t;
                }
            }
        }
        System.out.println("The sorted array is-\n");
        for(int x=0;x<n;x++)
        {
            System.out.println(a[x]+" ");  
        }
        
    }
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the size of the array\n");
        int n=sc.nextInt();
        int a[]=new int[n];
        for(int x=0;x<n;x++)
        {
            a[x]=sc.nextInt();
        }
        bubbleSort(a);
        
    }
}

File: bubblesort.c
/*Author : BRINDHA.V
  Date modified : 31.10.2021
*/


#include <stdio.h>
void bubblesort(int arr[], int size)
{
    int i, j;
    for (i = 0;  i < size; i++)
    {
        for (j = 0; j < size - i; j++)
        {
            if (arr[j] > arr[j+1])
                swap(&arr[j], &arr[j+1]);
 
        }
    }
}
void swap(int *a, int *b)
{
    int temp;
    temp = *a;
    *a = *b;
    *b = temp;
}
int main()
{
    int array[100], i, size;
    printf("How many numbers you want to sort:  ");
    scanf("%d", &size);
    printf("\nEnter %d numbers : ", size);
    for (i = 0; i < size; i++)
        scanf("%d", &array[i]);
    bubblesort(array, size);
    printf("\nSorted array is ");
 
    for (i = 0; i < size; i++)
        printf(" %d ", array[i]);
    return 0;
 
}
/*OUTPUT:

How many numbers you want to sort:  4

Enter 4 numbers : 4
3
2
1

Sorted array is  1  2  3  4 

...Program finished with exit code 0
Press ENTER to exit console.
*/